utils/subprocess_output.py

```diff
@@ -37,6 +37,10 @@ def get_subprocess_output(command, log, shell=False, stdin=None):
 
         stdout_f.seek(0)
         output = stdout_f.read()
+
+    if output_expected and output == "":
```

Comment:
nit: could change `output == ""` to `not output`. That way it'll catch `""` and `None`